### PR TITLE
Jadud predictable data

### PR DESCRIPTION
**Pull request by jadudm**
This merges in:

1. A renaming of files in the `init` directory, and 
2. IMLS-provided data that maps `fscs_id` and other information to be used in the front-end.

This most easily comes in at `init` time, as opposed to as a migration, because we really want to use `\copy` functionality native to Postgres to bring the CSV in.

### CODE REVIEW

File: github_conf/branch_protection_rules.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (cantsin) [inline github_conf/branch_protection_rules.json:1]: Why is this file here?

**Review by cantsin — APPROVED** [review]
One file i'm unsure of, the rest LGTM

File: github_conf/branch_protection_rules.json (hunk 1 above)
Comment (jadudm) [reply github_conf/branch_protection_rules.json]: I think this was resolved in Slack. It... was auto-generated? I know I did not explicitly create it. The suspect is `megalinter`.
